src/components/body/Body.tsx

```diff
@@ -0,0 +1,18 @@
+import "./body.css"  
+import Main from './main/Main'
+import Header from './header/Header'
+import Footer from "./footer/Footer"
+
+
+const Body: any = (props:any) => {
+    
+    return (
+        <div>
+            <Header/>
+            <Main/>
```

Comment:
please, rename `Main` to `Content`

Or - use another structure, for example:\
`<App/> --> <Content/> --> <Header/><Body/><Footer/>`

for example 2:\
`<App/> --> <Layout/> --> <Header/><Body/><Footer/>`

for example 3:\
`<App/> --> <Layout/> --> <Header/><Content/><Footer/>`

(it's better to use <Layout> I think)

Reply:
rename Main to Content